### DISCUSSION TITLE
ROCm 6.1.1 Release

### DISCUSSION THREAD

**Discussion by samjwu** [ROCm/ROCm#3102] [Announcements]
# ROCm 6.1.1 release notes

ROCm™ 6.1.1 introduces minor fixes and improvements to some tools and libraries.

## OS support

ROCm 6.1.1 has been tested against a pre-release version of Ubuntu 22.04.5 (kernel 5.15 [GA], 6.8 [HWE]).

## AMD SMI

AMD SMI for ROCm 6.1.1

### Additions

- Added deferred error correctable counts to `amd-smi metric -ecc -ecc-blocks`.

### Changes

- Updated the output of `amd-smi metric --ecc-blocks` to show counters available from blocks.
- Updated the output of `amd-smi metric --clock` to reflect each engine.
- Updated the output of `amd-smi topology --json` to align with output reported by host and guest systems.

### Fixes

- Fixed `amd-smi metric --clock`'s clock lock and deep sleep status.
- Fixed an issue that would cause an error when resetting non-AMD GPUs.
- Fixed `amd-smi metric --pcie` and `amdsmi_get_pcie_info()` when using RDNA3 (Navi 32 and Navi 31) hardware to prevent "UNKNOWN" reports.
- Fixed the output results of `amd-smi process` when getting processes running on a device.

### Removals

- Removed the `amdsmi_get_gpu_process_info` API from the Python library. It was removed from the C library in an earlier release.

### Known issues

- `amd-smi bad-pages` can result in a `ValueError: Null pointer access` error when using some PMU firmware versions.

```{note}
See the [detailed changelog](https://github.com/ROCm/amdsmi/blob/docs/6.1.1/CHANGELOG.md) with code samples for more information.
```

## HIPCC

HIPCC for ROCm 6.1.1

### Changes

- **Upcoming:** a future release will enable use of compiled binaries `hipcc.bin` and `hipconfig.bin` by default. No action is needed by users. You can continue calling high-level Perl scripts `hipcc` and `hipconfig`. `hipcc.bin` and `hipconfig.bin` will be invoked by the high-level Perl scripts. To revert to the previous behavior and invoke `hipcc.pl` and `hipconfig.pl`, set the `HIP_USE_PERL_SCRIPTS` environment variable to `1`.
- **Upcoming:** a subsequent release will remove high-level Perl scripts `hipcc` and `hipconfig`. This release will remove the `HIP_USE_PERL_SCRIPTS` environment variable. It will rename `hipcc.bin` and `hipconfig.bin` to `hipcc` and `hipconfig` respectively. No action is needed by the users. To revert to the previous behavior, invoke `hipcc.pl` and `hipconfig.pl` explicitly.
- **Upcoming:** a subsequent release will remove `hipcc.pl` and `hipconfig.pl`.

## ROCm SMI

ROCm SMI for ROCm 6.1.1

### Known issues

- ROCm SMI reports GPU utilization incorrectly for RDNA3 GPUs in some situations.

### hipBLASLt

hipBLASLt 0.7.0 for ROCm 6.1.1

#### Additions

- Added `hipblasltExtSoftmax` extension API.
- Added `hipblasltExtLayerNorm` extension API.
- Added `hipblasltExtAMax` extension API.
- Added `GemmTuning` extension parameter to set split-k by user.
- Added support for mixed precision datatype: fp16/fp8 in with fp16 outk.

##### Deprecations

- **Upcoming**: `algoGetHeuristic()` ext API for GroupGemm will be deprecated in a future release of hipBLASLt.

### hipSOLVER

hipSOLVER 2.1.1 for ROCm 6.1.1

#### Changes

- By default, `BUILD_WITH_SPARSE` is now set to OFF on Microsoft Windows.

#### Fixes

- Fixed benchmark client build when `BUILD_WITH_SPARSE` is OFF.

### rocFFT

rocFFT 1.0.27 for ROCm 6.1.1

#### Additions

- Enable multi-GPU testing on systems without direct GPU-interconnects.

#### Fixes

- Fixed kernel launch failure on execute of very large odd-length real-complex transforms.

<hr /><em>This discussion was created from the release <a href='https://github.com/ROCm/ROCm/releases/tag/rocm-6.1.1'>ROCm 6.1.1 Release</a>.</em>

**Comment by RomanAverin** [comment]
No consumer GPU support as before? For example the RX 7800 XT
